bootstrap.sh

```diff
@@ -45,20 +45,20 @@ then
 sudo mkdir /xDrip
 fi
 cd /xDrip
-sudo rm -rf scripts
-sudo -rf ConfigServer
+sudo rm -rf scripts # Delete all the scripts
```

Comment:
This comments are not saying anything that is not obvious.

Reply:
What's obvious to one, may not be obvious to everyone.
I would like to add more comments everywhere.

This particular line is the reason we had the recent issue.  It deletes lodash!  But, I ignored this and assumed that one can just run bootstrap.  The fact is that running bootstrap is very destructive.
I would like the destructive lines to stand out.

Reply:
Comments should say something that is not directly in the code.
See https://betterprogramming.pub/the-engineers-guide-to-writing-meaningful-code-comments-b8ab50b4ecfa for more explanations.
But since you are now the maintainer of the code, you can probably do whatever is needed.

Reply:
Thanks
I will read that carefully as I need a lot to learn.

I am not the maintainer of the code, I don't think.
I am a collaborator for the repository.
I will merge only the development branch only if you approve.

The stable branch (vps-1) will still be merged only by Jon.

Reply:
I see that my comments about deletions just state what the code does.
I am not sure what the very best practice should be.  I read that document and understand that we should not add noise to the code.  

But, when I look at Home.java in xDrip, as an example, I see a code that has more than 3800 lines.
There is no way I can read all of those 3800 lines.  But, when I see a comment in it, I know that a developer felt necessary to include a comment.  So, if I am trying to understand a part of that code, I will read the comments in that area.  And I am grateful to the developers who left those comments.

I added those deletions when I started working on making Nightscout on Google Cloud something that can be incrementally updated as opposed to something that could only be updated by deleting the virtual machine and starting from scratch.

But, there is a price to pay for making incremental updates possible and practical.   When I included in the guide that people could run a new bootstrap to switch to a different repository, I overlooked that I had added those deletions.
The reason I added those comments was that I was hoping no other developer would ever repeat that mistake.  But, perhaps that was a mistake and I need to accomplish that objective differently.
I'm not sure.